lib/task.js

```diff
@@ -99,6 +99,23 @@ async function buildVolumeBindings(taskVolumeBindings, volumeCache, taskScopes)
   return [caches, bindings];
 }
 
+function buildDeviceBindings(devices) {
+  let deviceBindings = [];
+  for (let deviceType in devices) {
+    let device = devices[deviceType];
+    device.mountPoints.forEach((mountPoint) => {
+      deviceBindings.push(
+        {
+          PathInContainer: mountPoint,
```

Comment:
Thought... specify the path that it should be mounted to within the container like we do for volume mounts.